utils/docker/images/Dockerfile.ubuntu-16.04

```diff
@@ -36,16 +36,42 @@
 
 # Pull base image
 FROM ubuntu:16.04
-MAINTAINER wojciech.uss@intel.com
+MAINTAINER marcin@slusarz@intel.com
```

Comment:
"marcin.slusarz@intel.com" instead of" marcin@slusarz@intel.com" ?

Reply:
true, fixed